### ISSUE TITLE
Push Access

### ISSUE THREAD

**Issue by KingHeavy** [Spearhead-Gaming-Milsim/Cav-Loadouts#1]
Requesting push access for cav loadouts

**Comment by coachwn** [comment]
You should already have access for that as a team owner in there....will check it shortly

**Comment by KingHeavy** [comment]
Yea, i don't be cause i can't upload files. I've got sqf's or sqm's or whatever we are calling them, set up for individuals. Also, side question, are we setting up these as our ideal western nato set up or what?

**Comment by coachwn** [comment]
yes ideal setup so that mission maker can use them as the base for whatever they are building. Send it to me here and I can upload. Not sure why you can't since I made you the admin on that team and repo
